### PR TITLE
[API compatibility] softmax, nonzero, randn

### PR DESCRIPTION
**Pull request by DongBaiYue**
### PR Category
Operator Mechanism

### PR Types
Improvements

### Description

This PR introduces PyTorch API compatibility aliases and resolves type checking issues:
1. Added PyTorch-compatible parameter aliases:

| API | param1 | alias1 | param2 | alias2 | 
|----------|----------|----------|----------|----------|
| [torch.nn.functional.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.softmax.html)  | x | input  | axis  | dim  | 
| [torch.nonzero](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.nonzero.html)  |  x | input  | | out |
| [torch.randn](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.randn.html)  |  shape | *size  |

Usage Example:
```
# Traditional Paddle call
output = paddle.rand([2,3])

# New PyTorch-compatible call
output = paddle.rand(2,3)
```

pcard-67164

### CODE REVIEW

**Review by zhangbo9674 — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM，中文文档同步修改
